encoding/config.go

```diff
@@ -57,6 +57,9 @@ func MakeConfig() ethermint.EncodingConfig {
 			"ethermint.evm.v1.MsgEthereumTx": customGetSignerFn("from"),
 		},
 	}
+	for k, v := range customGetSigners {
+		signingOptions.CustomGetSigners[k] = v
```

Comment:
Do we need this for Cronos's custom messages?

Reply:
I think so, from is different in [MsgTransferTokens](https://github.com/crypto-org-chain/cronos/blob/main/x/cronos/types/tx.pb.go#L90)

Reply:
It seems we just to put this in the msg proto to avoid custom get signers:

```
option (cosmos.msg.v1.signer) = "address";
```

Reply:
nice, I was lazy of resolving the dependencies